examples/transformation/REAMDE.rst

```diff
@@ -0,0 +1,29 @@
+.. _examples-transformation:
```

Comment:
Can you add this to `docs/source/usage/examples.rst` and maybe add a sub-section `Unit Tests` at the end?